src/bit_vec.rs

```diff
@@ -168,6 +210,7 @@ mod tests {
 	}
 
 	#[test]
+	#[cfg_attr(miri, ignore)] // BitVec error due to outdated version of bitvec
```

Comment:
the actuall error is: 
[full_error.txt](https://github.com/paritytech/parity-scale-codec/files/5731837/full_error.txt)

Once bitvec is migrated miri test failed at decoding because of wrong alignment. More precisely at line 67 `T::from_byte_slice(&vec)`
This is why I fixed it by copying from `Vec<u8>` to `Vec<T>`. This is not the fastest but at least it works.
Anyway I expect to have this refactored with upgrade of bitvec.

Some context on the alignment failure here https://github.com/brson/parity-scale-codec/commit/ad6c0aa00f8f503828a59a2f4a92cb8d5d7d280d#r45328246